testnets/nolustestnet/chain.json

```diff
@@ -6,7 +6,7 @@
   "network_type": "testnet",
   "pretty_name": "Nolus Testnet",
   "chain_type": "cosmos",
-  "chain_id": "rila-1",
+  "chain_id": "rila-3",
```

Comment:
If moving to a new chain, please close out the IBC connections that still exist for the rila-1 version of nolustestnet, or better yet, archive rila-1 to be nolustestnet1, and update the ibc connection files accordingly, and then rila-3 can take its place as 'nolustestnet'.